src/tools/plan-generator.ts

```diff
@@ -191,6 +191,7 @@ package ${this.namespace}
 // GENERATED CODE -- DO NOT EDIT
 //
 
+${tryImport('arcs.core.common.*', this.namespace)}
```

Comment:
Double check `arcs/sdk/BUILD`. Make sure that `arcs.core.common` is included in `exports`, or else this will fail. 

Ideally, import the specific class / function instead of all of core.common. Though, if you need multiple things, then the wildcard import is OK.

Reply:
Without the update, it may work in the open source and pass copybara's presubmit check, but will break G3 after.

Reply:
@cromwellian Does this sound right to you?

Reply:
I'll make sure this builds fully on g3

Reply:
It will only need it if the user actually needs to import the ArcId class into their file. They won't notice a strictdeps error in their IDE because they pick up the transitive dependency from the schema deps. 

I have a different objection, which is, don't use * imports. Emit the import you need. :)

Reply:
I normally don't use '.*' imports. I was following the convention in other generated code which seems to be doing so.

In any case, I think I'm going to step back and modify this change so that there's no need to mess with the code generators in this PR.